src/services/UserService.ts

```diff
@@ -326,6 +326,34 @@ const getUserSecurityGroup = async (userLoginId: string): Promise<any> => {
 
   return userSecurityGroup
 }
+const isUserFulfillmentAdmin = async (userLoginId: string): Promise<any> => {
+  const payload = {
+    inputFields: {
+      userLoginId,
+      permissionId: ['STOREFULFILLMENT_ADMIN']
+    },
+    entityName: "UserLoginSecurityGroupAndPermission",
+    filterByDate: "Y",
```

Comment:
Also we will get expired permission in this case. As the UserLoginSecurityGroupAndPermission view has dates for managing UserLogin and Security Group association (fromDate, thruDate) as well as dates for managing SecurityGroup and SecurityGroupPermission associations (SGPFromDate, SGPThruDate). The filterByDate parameter in performFind service by default consider fromDate and thruDate for filtering active records.

We need checks on both the dates which is not possible in performFind. You should use the getPermission api here which is used during login to fetch the user's permission.